ml-agents/mlagents/trainers/trainer/trainer.py

```diff
@@ -116,13 +241,39 @@ def save_model(self, name_behavior_id: str) -> None:
         """
         self.get_policy(name_behavior_id).save_model(self.get_step)
 
-    def export_model(self, name_behavior_id: str) -> None:
+    def export_model(self, name_behavior_id: str, is_checkpoint: bool = False) -> None:
```

Comment:
I still don't think we're 100% clear on where to use `name_behavior_id` vs `brain_name`.  Let's get that sorted before merging.

Reply:
`name_behavior_id` is required to fetch the policy. However, the trainer does save the attribute `brain_name` which is used to save the models, and that is exactly what we needed, solving the problem of using `name_behavior_id`.